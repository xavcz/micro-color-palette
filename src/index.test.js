const micro = require('micro');
const listen = require('test-listen');
const fetch = require('node-fetch');
const { describe, it, expect } = global;
const microColorPalette = require('./index');

describe('micro-color-palette', () => {
  it('works', async done => {
    const service = micro(microColorPalette);

    const url = await listen(service);
    const res = await fetch(url, {
      method: 'POST',
      body: JSON.stringify({ src: 'https://raw.githubusercontent.com/zeit/art/31913be3107827adf10e1f491ec61480f63e19af/micro/logo.png' }),
    });

    const palette = await res.json();

    expect(palette).toMatchSnapshot();
    done();
  });
});
