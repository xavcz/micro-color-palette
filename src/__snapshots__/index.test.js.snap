// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`micro-color-palette works 1`] = `
Array [
  "#3494fc",
  "#64acfc",
  "#0474fc",
  "#8cc4fc",
  "#3cb0fc",
]
`;
